spec/requests/api/v1/companies_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/v1/companies', type: :request do
+
+  before { create_logged_in_user }
+
+  path '/api/v1/users' do
+    
+    get('list users') do
```

Comment:
The way it is now, the endpoint request only works if you login in punch before. Another possible approach is to pass the token as a parameter(maybe in header or in query), it will look something like this: 
`get('list users') do
      parameter name: :token, in: :header, type: :string`